src/config.rs

```diff
@@ -22,9 +22,6 @@ pub const DEFAULT_ZOME_CALL_TIMEOUT: std::time::Duration = std::time::Duration::
 /// Errors when parsing config arguments.
 #[derive(Debug, thiserror::Error)]
 pub enum ConfigParseError {
-    /// Error when parsing URL.
-    #[error("URL parse error: {0}")]
-    UrlParseError(#[from] url2::Url2Error),
```

Comment:
This was such a pain to work with. Within the lib it is now only a `SocketAddr`. That makes setting up app connections to the same IP really simple and avoids having to extract parts of the URL in multiple places.